includes/Admin/Orders.php

```diff
@@ -308,6 +308,8 @@ public function maybe_stop_order_email( $is_enabled, $order ) {
 	 */
 	public function is_order_editable( $maybe_editable, \WC_Order $order ) {
```

Comment:
```suggestion
	public function is_order_editable( $maybe_editable, $order ) {
```

I realized that this is a direct hook callback, so we can't enforce a type here. I think we'll need to check that `$order instanced \WC_Order` within the method.

Reply:
Good point, @ChaseWiseman.

https://github.com/facebookincubator/facebook-for-woocommerce/pull/1586/commits/d9cd5c36fbbe84143c34c174525836733fb4c507 and https://github.com/facebookincubator/facebook-for-woocommerce/pull/1586/commits/f5b70b88dcb7bafd7b897f317d5ca69fde3d8a26 will fix that.

Note: https://github.com/facebookincubator/facebook-for-woocommerce/pull/1586/commits/f5b70b88dcb7bafd7b897f317d5ca69fde3d8a26 determines whether `$order` is an instance of `\WC_Order` and if it's pending. This method body could be reduced to a single `return` conditional line, however, to make it readable, I've introduced two vars. If a better practice for this project suggests that it should be reduced as much as possible, let me know and I'll submit another commit with that.